README.md

```diff
@@ -101,6 +103,8 @@ Your import statement should now look like this:
 
 ```go
 import (
+    "context"
```

Comment:
Looks like there are other missing imports here, too. Can you also add `"log"` and `"time"` after `"context"`?